helpers/json.js

```diff
@@ -0,0 +1,4 @@
+module.exports = data => (req, res) => {
+  res.setHeader('content-type', 'application/json');
+  res.end(JSON.stringify(data));
```

Comment:
could not be sure if we need a guard in case of some data comes which is not proper json (that will cause a js error somehow)

Reply:
It's stringify right? I think it can handle almost anything...

Reply:
I think, there should be no problem for that.